vscode-lean4/src/utils/leanInstaller.ts

```diff
@@ -62,9 +58,6 @@ export class LeanInstaller implements Disposable {
                 // Ah, there is no elan, but what if Lean is in the PATH due to custom install?
                 const found = await this.checkLeanVersion(packageUri, leanVersion);
                 if (found.error) {
-                    // Ah, then we need to install elan so prompt the user, but don't wait
-                    // here for the answer because that could be forever.
-                    void this.showInstallOptions(packageUri);
```

Comment:
not waiting on a window.showErrorMessage is bogus and should never be allowed, because it means the popup window can randomly disappear such that the user might not ever see it.